dipy/tracking/streamline.py

```diff
@@ -469,6 +470,83 @@ def select_by_rois(streamlines, rois, include, mode=None, affine=None,
             yield sl
 
 
+def cluster_confidence(streamlines, max_mdf=5, subsample=12, power=1,
+                       override=False):
+    """ Computes the cluster confidence index (cci), which is an
+    estimation of the support a set of streamlines gives to
+    a particular pathway.
+
+    Ex: A single streamline with no others in the dataset
+    following a similar pathway has a low cci. A streamline
+    in a bundle of 100 streamlines that follow similar
+    pathways has a high cci.
+
+    See: Jordan et al. 2017
+    (Based on streamline MDF distance from Garyfallidis et al. 2012)
+
+    Parameters
+    ----------
+    streamlines : list of 2D (N, 3) arrays
+        A sequence of streamlines of length N (# streamlines)
+    max_mdf : int
+        The maximum MDF distance (mm) that will be considered a
+        "supporting" streamline and included in cci calculation
+    subsample: int
+        The number of points that are considered for each streamline
+        in the calculation. To save on calculation time, each
+        streamline is subsampled to subsampleN points.
+    power: int
+        The power to which the MDF distance for each streamline
+        will be raised to determine how much it contributes to
+        the cci. High values of power make the contribution value
+        degrade much faster. Example: a streamline with 5mm MDF
+        similarity contributes 1/5 to the cci if power is 1, but
+        only contributes 1/5^2 = 1/25 if power is 2.
+    override: bool, False by default
+        override means that the cci calculation will still occur even
+        though there are short streamlines in the dataset that may alter
+        expected behaviour.
+
+    Returns
+    -------
+    Returns an array of CCI scores
+
+    References
+    ----------
+    [Jordan17] Jordan K. Et al., Cluster Confidence Index: A Streamline‐Wise
```

Comment:
Something about this reference is wonky and throwing 27 off. Could you rewrite it from scratch by typing it manually into the file, instead of copy-paste?

Reply:
thanks, @arokem. There was a rogue dash, but I replaced it and then 27 passed.

Reply:
🎉